op-wheel/commands.go

```diff
@@ -59,12 +60,16 @@ var (
 		Value:    "http://localhost:8551/",
 		EnvVars:  prefixEnvVars("ENGINE"),
 	}
+	EngineJWT = &cli.StringFlag{
+		Name:    "engine.jwt-secret",
+		Usage:   "JWT secret used to authenticate Engine API communication with.",
```

Comment:
unlikely to be a problem, but might be worth mentioning that this new flag takes precedence of the file